### PR TITLE
Create SimpleExport.pyde

### PR DESCRIPTION
**Pull request by villares**

### CODE REVIEW

File: examples.py/Library/DXF Export/SimpleExport.pyde [hunk 1]
```diff
@@ -0,0 +1,40 @@
+""" Simple DXF Export
+    by Simon Greenwold.
+    
+    Press the 'R' key to export a DXF file.
+    """
```
Comment (jdf) [inline examples.py/Library/DXF Export/SimpleExport.pyde]: Maybe auto-format this file?

Comment (villares) [inline examples.py/Library/DXF Export/SimpleExport.pyde]: Sure!
(very new to this, can you see now range(-2, 2) instead of range(-2,2)?

Comment (jdf) [inline examples.py/Library/DXF Export/SimpleExport.pyde]: Please format the docstring per

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

**Comment by jdf** [comment]
Thank you.

**Pull request merged**